tests/dispense.rs

```diff
@@ -226,42 +209,11 @@ async fn _dispense_sends_coins_to_valid_address(
     assert_eq!(test_balance, context.faucet_config.dispense_amount);
 }
 
-fn generate_recipient_addresses(count: usize, rng: &mut StdRng) -> Vec<String> {
-    let recipient_addresses: Vec<Address> =
-        std::iter::repeat_with(|| rng.gen()).take(count).collect();
-    recipient_addresses
-        .iter()
-        .map(|addr| format!("{}", addr))
-        .collect()
-}
-
 #[tokio::test]
 async fn many_concurrent_requests() {
```

Comment:
This test verifies concurrent transaction logic. The idea is to test that requesting funds for 30 different accounts will produce 30 transactions on the blockchain side. I don't see how it can be achieved now=)